doc/user/content/sql/drop-index.md

```diff
@@ -61,21 +51,28 @@ You can remove an index with any of the following commands:
   DROP INDEX q01_geo_idx CASCADE;
   ```
 
-### Do not issue an error if attempting to remove a nonexistent index
+If the index `q01_geo_idx` does not exist, the above operations return an error.
 
-```mzsql
-DROP INDEX IF EXISTS q01_geo_idx;
-```
+{{< tip >}}
 
-## Privileges
+In the **Materialize Console**, you can view existing indexes in the [**Database
+object explorer**](/console/data/). Alternatively, you can use the
+[`SHOW INDEXES`](/sql/show-indexes) command.
 
-The privileges required to execute this statement are:
+{{< /tip >}}
```

Comment:
Could we move this to the top (_i.e._, before showing the `DROP` command)? Also, I'd remove the alternative syntaxes from the examples, since they all have the same effect. Just adds overhead, and we already note they're all the same earlier.

Reply:
Will do!